content/docs/command-reference/remote/modify.md

```diff
@@ -288,23 +288,24 @@ For more information about the variables DVC supports, please visit
 
 <details>
 
-### Click for S3 API compatible storage
+### Click for S3-compatible storage
 
-To communicate with a remote object storage that supports an S3 compatible API
-(e.g. [Minio](https://min.io/),
-[DigitalOcean Spaces](https://www.digitalocean.com/products/spaces/),
-[IBM Cloud Object Storage](https://www.ibm.com/cloud/object-storage) etc.),
-configure the remote's `endpointurl` explicitly:
+To communicate with a remote object storage that supports an S3-compatible API,
+configure the remote's `endpointurl`:
 
 ```dvc
 $ dvc remote add -d myremote s3://mybucket/path
 $ dvc remote modify myremote endpointurl \
                     https://object-storage.example.com
 ```
 
-Besides that, any parameters that are available for Amazon S3 (see previous
-section) may be available for S3 compatible storage. For example, let's setup a
-DVC remote using the `example-name`
+It's also important to setup appropriate authentication with any of the related
```

Comment:
How about: 

⚠️  It's important to setup appropriate authentication with `dvc remote modify`. Otherwise, DVC uses the default AWS credentials which may cause an error.

Reply:
Moved to https://github.com/iterative/dvc.org/pull/2415#pullrequestreview-644086184.